vertx-mongo-client/src/main/asciidoc/java/index.adoc

```diff
@@ -167,6 +167,26 @@ mongoClient.insert("books", document, res -> {
 });
 ----
 
+To insert a documents you use `link:../../apidocs/io/vertx/ext/mongo/MongoClient.html#insertMany-java.lang.String-java.util.List-io.vertx.core.Handler-[insertMany]`.
```

Comment:
This file is auto-generated when you build the project from package-info.java.

You should add the documentation in package-info.java.

Also please make sure you build and test the project before you push.

mvn clean install